internal/measurement/measurement.go

```diff
@@ -172,69 +176,55 @@ func Percentage(value, total int64) string {
 	}
 }
 
+// unit includes a list of names representing a specific unit and a factor
+// which one can multiple a value in the specified unit by to get the value
+// in terms of the base unit.
+type unit struct {
+	preferredName string
```

Comment:
Maybe "canonical" instead of "preferred"? Not a strong opinion.

Reply:
Done and thanks! Canonical reads much better to me.